src/App.tsx

```diff
@@ -13,20 +13,23 @@ import Accessibility from './screens/Accessibility'
 import ContextProvider from './state/ContextProvider'
 
 function App() {
-  const { contentFontFamily, actionsFontFamily } = useContext(ContextProvider)
+  const { epic, gog, contentFontFamily, actionsFontFamily } =
+    useContext(ContextProvider)
 
   const style = {
     '--content-font-family': contentFontFamily,
     '--actions-font-family': actionsFontFamily
   } as React.CSSProperties
 
+  const loggedIn = epic.username || gog.username
+
   return (
     <div className="App" style={style}>
       <HashRouter>
         <Sidebar />
         <main className="content">
           <Routes>
-            <Route path="/" element={<Library />} />
+            <Route path="/" element={loggedIn ? <Library /> : <Login />} />
```

Comment:
handle the required login here instead of in SidebarLinks, not sure if we want to require user login for other urls, maybe Settings should have the same?